CMakeLists.txt

```
@@ -991,9 +992,20 @@ if(WITH_OPTIM)
                 set(WITH_SSSE3 OFF)
             endif()
         endif()
+        if(WITH_SSE41)
+            check_sse41_intrinsics()
+            if(HAVE_SSE41_INTRIN AND WITH_SSE41)
```

SSE4.1 is enabled if SSSE3 is present:

```suggestion
            if(HAVE_SSE41_INTRIN AND WITH_SSSE3)
```